types/ol/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
```

Comment:
For new packages, it should only extend `dtslint/dt.json`, no other rules

Reply:
thanks!